### PR TITLE
Bump up formik to 2.2.9

### PR DESCRIPTION
**Pull request by Zangetsu101**

### CODE REVIEW

File: packages/client/src/components/form/FormFieldGenerator.tsx [hunk 1]
```diff
@@ -817,30 +815,10 @@ class FormSectionComponent extends React.Component<Props> {
...
-     * }
-     *
-     * 22.8.2019
-     *
-     * This might be because of setState not used with the function syntax
-     */
-
-    const fieldsWithValuesDefined = fields.filter(
-      (field) => values[field.name] !== undefined
-    )
```
Comment (Zangetsu101) [inline packages/client/src/components/form/FormFieldGenerator.tsx:839]: This issue is resolved after the changes

Comment (Zangetsu101) [inline packages/client/src/components/form/FormFieldGenerator.tsx:839]: This issue is resolved after the changes

File: packages/client/src/forms/certificate/fieldDefinitions/collectorSection.ts [hunk 2]
```diff
@@ -389,8 +389,7 @@ export const collectBirthCertificateFormSection: IFormSection = {
           conditionals: [
             {
               action: 'hide',
-              expression:
-                'draftData?.registration?.certificates?.length && draftData?.registration?.certificates[0]?.collector?.noAffidavitAgreement?.length !== 0'
+              expression: 'values.noAffidavitAgreement?.length !== 0'
```
Comment (euanmillar) [inline packages/client/src/forms/certificate/fieldDefinitions/collectorSection.ts:392]: @Zangetsu101 I love it!

**Review by euanmillar — APPROVED** [review]
Great work!

**Pull request merged**